### ISSUE TITLE
Pretrain feature extractor

### ISSUE THREAD

**Issue by yjhong89** [tianyeli/tofu#2]
Dear author.

I am YJHong and researching on vertex refinement.

Thanks for your great works!

Since the code is not released yet, I have few questions about implementation.

1. How pretrain feature extractor (UNet with ResNet34 backbone) ?
    - which datasets and loss function ?
    - is it freezed when training V2V ?
2. How finetune to new datasets (case of generalization to CoMA dataset)
    - Finetune whole architecture including global and local stage ?
    - If it were true, how set ground truth vertices for each stage ? (The number of CoMA dataset vertices is different to ToFu)
    
Thank you!
YJHong.
